tests/worker/file_util_test.py

```diff
@@ -65,3 +65,24 @@ def test_bz2_file(self):
 
     def test_gzip_bytestring(self):
         self.assertEqual(un_gzip_bytestring(gzip_bytestring(b'contents')), b'contents')
+
+    def test_tar_exclude_ignore(self):
+        dir = os.path.join(
+            os.path.dirname(os.path.dirname(__file__)), 'files/dirToTestExcludeIgnore'
```

Comment:
In general, can we make these directory names less Java-like?  (i) shorter and (ii) using snake_case...  I think 'ignore_test' would suffice.